### PR TITLE
Bounty row layout fix(es)

### LINKED ISSUE #2367: bug - padding on /explorer and bounty
was the padding on these pages changed intentionally?

screen shot 2018-10-05 at 1 57 39 pm
screen shot 2018-10-05 at 1 57 58 pm

looked much better before IMHO

for reference, this is what it looked like before (and still looks like in stage ) 
screen shot 2018-10-05 at 1 58 13 pm

### PR DESCRIPTION
**Pull request by pinkiebell**
Fixes #2367

### CODE REVIEW

**Comment by owocki** [comment]
yay thank you!

File: app/assets/v2/css/dashboard.css [hunk 1]
```diff
@@ -75,6 +75,7 @@ body {
   display: flex;
   margin-top: 2em;
   margin-right: 8px;
+  padding-left: 8px !important;
```
Comment (thelostone-mc) [inline app/assets/v2/css/dashboard.css]: Can we do without using `!important` ?

Comment (pinkiebell) [reply app/assets/v2/css/dashboard.css]: @thelostone-mc 
Thanks for the reminder! 😄

**Comment by owocki** [comment]
@thelostone-mc @mbeacom ok to 🚢 ?

**Pull request merged**